javalib/src/main/scala/java/lang/Thread.scala

```diff
@@ -548,10 +564,10 @@ object Thread {
   def getAllStackTraces(): java.util.Map[Thread, Array[StackTraceElement]] =
     throw new UnsupportedOperationException()
 
-  private var defaultExceptionHandler: UncaughtExceptionHandler = _
+  @volatile private var defaultExceptionHandler: UncaughtExceptionHandler = _
   def getDefaultUncaughtExceptionHandler(): UncaughtExceptionHandler =
     defaultExceptionHandler
-  def setDefaultUncaughtHandler(eh: UncaughtExceptionHandler): Unit =
+  def setDefaultUncaughtExceptionHandler(eh: UncaughtExceptionHandler): Unit =
```

Comment:
Maybe a silly question, who calls `setDefaultUncaughtExceptionHandler`? I couldn't find the caller of this method in this project, and `ThreadGroup#uncaughtException` always goes this branch https://github.com/natsukagami/scala-native/blob/ac2460c1fa20081ca82a18cc564e7686136a17a9/javalib/src/main/scala/java/lang/ThreadGroup.scala#L222-L225 ?